### ISSUE TITLE
Change button layout to 3 across on location pages

### ISSUE THREAD

**Issue by karmenc** [massgov/feature-requests#256]
Currently, buttons go 2 across, which leaves a lot of white space on the right and adds to the overall page length (example: https://www.mass.gov/locations/attleboro-district-court). Changing the button layout to 3 across would make better use of the spacing on the page.

**Comment by joeg8612** [comment]
I don't think we ever envisioned that container to have more than one or two quick links. I will discuss this with the content strategists

**Comment by joeg8612** [comment]
This was implemented.
